app/presenters/property_presenter.rb

```diff
@@ -40,39 +46,14 @@ def property_value
     property[1]
   end
 
-  def render_scripts_table(entry)
-    thead = content_tag(:thead) do
-      content_tag(:tr) do
-        content_tag(:th, 'id').
-          concat(content_tag(:th, 'output'))
-      end
-    end
-    tbody = content_tag(:tbody) do
-      entry[:scripts].map do |script|
-        content_tag(:tr) do
-          content_tag(:td, script[0]).
-            concat content_tag(:td, content_tag(:pre, script[1]))
-        end
-      end.join.html_safe
-    end
-
-    # content_tag(:div,
-    #   content_tag(:table, thead.concat(tbody)),
-    #   class: 'content-textile'
-    # )
-    content_tag(:table, thead.concat(tbody), class: 'table table-condensed')
-  end
-
   def render_table
     values         = property_value
-    column_names   = values.map(&:keys).flatten.uniq.sort.map(&:to_sym)
-    sorted_entries = column_names.include?(:port) ? values.sort_by{ |h| h[:port] } : values
-
-    output = if column_names.delete(:scripts)
-               render_tabs(sorted_entries)
-             else
-               ''
-             end
+    column_names   = values.map(&:keys).flatten.uniq.sort.map(&:to_sym).
+                     delete_if{ |cn| !services_table_columns.include?(cn) }
```

Comment:
this line is from @dormi's original commits. But isn't `render_table` going to be used for other tables, not just for the Services table? It's called whenever `property_value` is an array of hashes - but with this line (unless I'm misunderstanding something), the table is going to be empty whenever none of the columns are in `services_table_columns` (which is likely going to be any table other than services).

Do we need to handle the case where the property is an array of hashes, but isn't `:services` or `:services_extras`? If so, shouldn't this line be removed? If not, `render_table` can be removed entirely.

Reply:
> But isn't render_table going to be used for other tables, not just for the Services table?

That's the idea, yes.

Right now, this rendering code is failing for a property like:

```
...
"foo": [
{ "port": "80", "state": "open" },
{ "port": "443", "state": "open" },
{ "port": "25", "state": "open" }
],
...
```

When a simple table should be shown.